packages/es-components/src/components/controls/buttons/DropdownButton.js

```diff
@@ -59,6 +138,10 @@ export class DropdownButton extends React.Component {
     this.triggerButton = ref;
   };
 
+  setButtonDropdownRef = ref => {
+    this.buttonDropdown = ref;
```

Comment:
I can't wait to replace this with `createRef` when we start supporting `16.3.0` and up only.